research-hub-web/src/app/app.component.ts

```diff
@@ -135,31 +230,33 @@ export class AppComponent implements OnInit, OnDestroy, AfterViewInit {
     this.appComponentService.setContentSidenavHasContent(hasContent);
   }
 
-  /**
-   * Sets the page title, and enables/disables the search bar and header.
-   * Also sets any custom CSS for the page.
-   * @param pageInfo currentPage information object.
-   */
-  setTitleSearchBarHeaderCustomCSS(pageInfo: any) {
-    if (pageInfo.title) {
-      this.titleService.setTitle('ResearchHub: ' + this.pageTitle);
-    }
-    this.headerService.setBatchParams(pageInfo.title, pageInfo.description, pageInfo.imageUrl, pageInfo.isHeaderVisible);
-    this.searchBarService.setVisibility(pageInfo.isSearchBarVisible);
-    this.appComponentService.setCustomCSSClassName(pageInfo.customCSSClassName);
-  }
+  async ngOnInit() {  
+    this.title = "Welcome to ResearchHub"
+    this.summary = "The ResearchHub connects you with people, resources, and services from across the University to enhance and accelerate your research."
+
+    // Methods dependent on URL
+    this.url = this.appComponentService.url.subscribe(url => {
+      if (url == 'home') {
+        this.titleService.setTitle('ResearchHub | Home');
+        this.showBanner = true;
+      }
+      else {
+        this.titleService.setTitle('ResearchHub | Home');
+        this.showBanner = false;
+      }
+      this.searchBarService.setVisibility(url == 'home' || url.substring(0, 6) == 'search' ? true : false);
+    });
 
-  async ngOnInit() {
     this.titleSub = this.appComponentService.titleChange.subscribe((title) => {
       this.pageTitle = title;
-      this.setTitleSearchBarHeaderCustomCSS(this.optionsService.getPageInfo(this.currentRoute, this.pageTitle));
+      this.titleService.setTitle('ResearchHub | ' + this.pageTitle);
     });
```

Comment:
I would delete all of this:

```ts
    this.url = this.appComponentService.url.subscribe(url => {
      if (url == 'home') {
        this.titleService.setTitle('ResearchHub | Home');
        this.showBanner = true;
      }
      else {
        this.titleService.setTitle('ResearchHub | Home');
        this.showBanner = false;
      }
      this.searchBarService.setVisibility(url == 'home' || url.substring(0, 6) == 'search' ? true : false);
    });
```

And just leave this:
```ts
    this.titleSub = this.appComponentService.titleChange.subscribe((title) => {
      this.pageTitle = title;
      this.titleService.setTitle('ResearchHub | ' + this.pageTitle);
    })
```

Then change the the initial `titleChange` service from a `Subject` to a `BehaviourSubject` and give it a default value:

```ts
  public titleChange: BehaviorSubject<string> = new BehaviorSubject<string>('Home');
```

This will also make us stop being lazy devs/not setting titles for new pages :v: